modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/nhttp/HttpCoreNIOSender.java

```diff
@@ -153,12 +153,9 @@ public void init(ConfigurationContext cfgCtx, TransportOutDescription transportO
 
         connpool = new ConnectionPool();
 
-        proxyConfig = new ProxyConfigBuilder().parse(transportOut).build();
-        if (log.isInfoEnabled() && proxyConfig.getProxy() != null) {
-            log.info("HTTP Sender using Proxy " + proxyConfig.getProxy() + " bypassing " + 
-                proxyConfig.getProxyBypass());
-        }
-        
+        proxyConfig = new ProxyConfigBuilder().build(transportOut);
+        log.info(proxyConfig.logProxyConfig());
+
         Parameter param = transportOut.getParameter("warnOnHTTP500");
```

Comment:
You should probably introduce an appropriate constant upon the parameter name "warnOnHTTP500".